tests/test_util.py

```diff
@@ -3,15 +3,27 @@
 
 import contextlib
 import functools
+import os
+import subprocess
 import zipfile
 from hashlib import sha1
+from textwrap import dedent
 
-from twitter.common.contextutil import temporary_file
+import pkg_resources
+from twitter.common.contextutil import temporary_dir, temporary_file
 
+from pex.common import safe_mkdir
+from pex.compatibility import nested
 from pex.installer import EggInstaller, WheelInstaller
+from pex.pex_builder import PEXBuilder
 from pex.testing import make_bdist, temporary_content, write_zipfile
 from pex.util import CacheHelper, DistributionHelper
 
+try:
+  import mock
+except ImportError:
+  from unittest import mock
```

Comment:
should be the other way around due to http://code.google.com/p/mock/issues/detail?id=249